servers/rendering_server.h

```diff
@@ -72,6 +72,10 @@ class RenderingServer : public Object {
 public:
 	static RenderingServer *get_singleton();
 	static RenderingServer *create();
+	static Vector2 _norm_to_oct(const Vector3 v);
```

Comment:
I actually think that there's a new function in Vector3 which also does octahedral encoding - it might be good to consolidate those implementations (although I think that implementation maps it to UNORM rather than SNORM as we do here but that's not difficult to figure out, would just take some additional changes elsewhere to support UNORM values)

Reply:
I went ahead and implemented things using the existing octahedron implementation for both the normals and tangents! I added the tangent-specific functions to Vector3 as well

As for decoding Snorm vs Unorm, the only thing that really changes is that we have to do two additional Multiply/Add operations in the shaders to decode the normals back to cartesian, not sure how sensitive we are to the performance implications, but wanted to note it at least

let me know what you all think!